src/components/Dashboard/Content/ContentList/index.jsx

```diff
@@ -54,24 +54,26 @@ function ContentList() {
   return (
     <>
       <VideoModal searchResults={searchResults} />
-      <ComponentLayout title="Videos Page" heading="Videos">
-        <div className="contents-list-container__top-sub-container">
-          <AutoComplete
-            value={value}
-            options={contentsTitles}
-            onSelect={onSelect}
-            onChange={onChange}
-            onInputKeyDown={onInputKeyDown}
-            filterOption={filterOptions}
-            placeholder="Search a video"
-            style={{
-              width: 350,
-            }}
-          />
-        </div>
-        <div className="contents-list-bottom">
-          <div className="contents-list">
-            {
+      {
+      contentsCount > 0 ? (
```

Comment:
```suggestion
      contentsCount  ? (
```
suppose to do the trick too.
cuz if it equals to `0`, it's a `falsy` value, hence the execution will go directly to the `else` statement.
- What do we mean by `falsy` value?
- any option among these `false, 0, -0, "", '', ``, null, undefined, NAN`.
[see these examples](https://www.sitepoint.com/javascript-truthy-falsy/)